tests/laser/keccak_tests.py

```diff
@@ -0,0 +1,113 @@
+from mythril.laser.smt import Solver, symbol_factory, And
+from mythril.laser.ethereum.keccak_function_manager import keccak_function_manager
+import z3
+import pytest
+
+
+@pytest.mark.parametrize(
+    "input1, input2, expected",
```

Comment:
Would it make sense to have an additional case where one of the bitvecs is symbolic and the other not.
They can be equal if the concrete value is unconstrained. Otherwise probably not